components/p3a/p3a_message.cc

```diff
@@ -128,7 +128,7 @@ std::string GenerateP3AConstellationMessage(base::StringPiece metric_name,
                  });
 
   return base::JoinString(serialized_attributes,
-                          kP3AMessageConstellationKeyValueSeparator);
+                          kP3AMessageConstellationLayerSeparator);
```

Comment:
having a unit-test would be great

Reply:
just amended the test util. should suffice